server/controllers/connectedAccounts.js

```diff
@@ -47,7 +48,25 @@ module.exports = (app) => {
       } else {
         return next(new errors.BadRequest('Github authorization failed'));
       }
-    }
+    },
 
+    fetchAllRepositories: (req, res, next) => {
+      const payload = req.jwtPayload;
+      ConnectedAccount
+      .findOne({where: {id: payload.connectedAccountId}})
+      .then(ca => {
+        const options = {
+          url: `https://api.github.com/user/repos?per_page=1000&sort=stars&access_token=${ca.secret}&type=all`,
+          headers: {
+            'User-Agent': 'OpenCollective'
+          },
+          json: true
+        };
+        request(options, (error, response, body) => {
```

Comment:
promise chain is broken here, need to return, e.g cfr other comment: `return Promise.promisify(request, {multiArgs: true})(options).then(args => args[1])`

Then adding at the end of the promise chain:
```
  .then(body => res.json(body))
  .catch(next)
```